0.5/articles/unit-testing-elements.md

```diff
@@ -269,7 +269,7 @@ Give it an ID so we can easily reference it amongst other instances on the page.
 Next, we'll query the DOM for the "selector1" element we just included included:
 
     <script>
-      var s = document.querySelector('#selector1');
+      var s = document.querySelector('#selector1'); // why not byid?
```

Comment:
I've seen this benchmarked and iirc the performance difference is significant....though, I imagine those tests are for large(r) documents. Perhaps it doesn't make any difference in smaller (shadow) DOMs? It'd be nice to know, so we don't have to be concerned...still, a side-issue for this doc, imo.